ccd-definition/ComplexTypes.json

```diff
@@ -314,7 +314,7 @@
     "LiveFrom": "01/01/2017",
     "ID": "Applicant",
     "ListElementCode": "address",
-    "FieldType": "TextArea",
+    "FieldType": "AddressUK",
     "ElementLabel": "Address",
```

Comment:
Current Address in other places, shoudl line 318 also be Current Address?

Reply:
No, for applicant it should read 'Address'. 'Current address' elsewhere.